### DISCUSSION TITLE
java运行环境需求建议

### DISCUSSION THREAD

**Discussion by LyingDoc** [1Panel-dev/1Panel#5831] [Q&A]
### 1Panel 版本

1.11.12beat

### 请描述您的需求或者改进建议

1.可以内部端口访问，我测试无法进行内部端口访问，必须开启外部端口访问
2.既然在运行环境就已经启动了jar包，为啥还可以创建go，java等运行环境，这意义在哪里
以上可以参考宝塔
这个地方构建环境为啥还要让人启动啊
这是宝塔的，你可以合并到一起，选安装jdk的包不让在构建环境就启动起来，还要手动找目录

**Comment by zhengkunwang223** [comment]
1. 提供一下具体日志
2. 没看明白
